src/staging/bot.py

```diff
@@ -1516,7 +1516,7 @@ async def configure_devel_bci_package(self, package_name: str) -> None:
             )
 
         assert (
-            len(bci) == 1
+            len(bci) >= 1
         ), f"Got {len(bci)} packages with the name {package_name}: {bci}"
```

Comment:
The message can now be made more specific: 
```suggestion
        ), f"Got no packages with the name {package_name}"
```